packages/angular/pwa/pwa/index.ts

```diff
@@ -38,18 +37,8 @@ function addServiceWorker(options: PwaOptions): Rule {
 }
 
 function getIndent(text: string): string {
-  let indent = '';
-  let hitNonSpace = false;
-  text.split('')
-    .forEach(char => {
-      if (char === ' ' && !hitNonSpace) {
-        indent += ' ';
-      } else {
-        hitNonSpace = true;
-      }
-    }, 0);
-
-  return indent;
+  // This RegExp is guaranteed to match any string (even if it is a zero-length match).
+  return (/^ */.exec(text) as RegExpExecArray)[0];
```

Comment:
What about a quick loop instead of the regex (plus tab support):
```
let indent = '';
for (const char of text) {
    if (char === ' ' || char === '\u0009') {
        indent += char;
    } else {
        break;
    }
}
```

Reply:
I personally like RegExps, but happy to change it to whatever you prefer (or even keep the original version).
I just happened to go through the files trying to fix something else and made some changes that made sense to me (and hopefully to others).

Reply:
I'm partial to the more self explanatory nature of the loop above then the other two options.  And it definitely performs better than the first.

Reply:
To each his own 😛
Happy to change it to a `for` loop.